src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacKeyFieldDeclaration/KeyFieldDacTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.DacExtensionDefaultAttribute;
+using Acuminator.Analyzers.StaticAnalysis.DacKeyFieldDeclaration;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.DacKeyFieldDeclaration
+{
+	public class KeyFieldInDacTests : CodeFixVerifier
+	{
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new KeyFieldDeclarationAnalyzer();
+
+		protected override CodeFixProvider GetCSharpCodeFixProvider() => new KeyFieldDeclarationFix();
+
+		[Theory]
+		[EmbeddedFileData("CompoundKey.cs")]
+		public virtual void TestCompoundKey(string source) =>
+			VerifyCSharpDiagnostic(source);
+
+		[Theory]
+		[EmbeddedFileData("Key.cs")]
+		public virtual void TestKey(string source) =>
+			VerifyCSharpDiagnostic(source);
+
+		[Theory]
+		[EmbeddedFileData("IdentityKey.cs")]
+		public virtual void TestIdentityKey(string source) =>
+			VerifyCSharpDiagnostic(source);
+
+		[Theory]
+		[EmbeddedFileData("IdentityKey+Key.cs")]
+		public virtual void TestIdentityKey_Key(string source) =>
+			VerifyCSharpDiagnostic(source,
+				Descriptors.PX1055_DacKeyFieldBound.CreateFor(locations: new (int line, int column)[] { (line: 10, column: 4), (line: 10, column: 4), (line: 16, column: 4) }),
```

Comment:
Formatting: it is impossible to read such long lines

Reply:
@saloponov please use the same format for such methods as we use in other unit tests.